### PR TITLE
Upgrade pipelines file generation to v2.2

### PR DESCRIPTION
**Pull request by lyuboxa**
### Description

* Remove `processors.Type`
* Validate `processors.Plugin` is not empty

### Quick checks:

- [x] I have followed the [Code Guidelines](https://github.com/ConduitIO/conduit/blob/main/docs/code_guidelines.md).
- [x] There is no other [pull request](https://github.com/conduitio/conduit-operator/pulls) for the same update/change.
- [ ] I have written unit tests.
- [x] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

File: api/v1alpha/conduit_validator.go [hunk 1]
```diff
@@ -1,18 +1,32 @@
...
 }
 
-func ValidatePluginType(c *ConduitConnector) error {
+func validateConnectorPluginType(c *ConduitConnector) error {
 	return conduit.ValidatePluginType(c.Type)
 }
+
+func validateProcessorPlugin(p *ConduitProcessor) error {
+	if p.Plugin == "" {
+		return fmt.Errorf("plugin %q is invalid", p.Plugin)
```
Comment (hariso) [inline api/v1alpha/conduit_validator.go:28]: Nitpick: it would be nice if the error was more specific, e.g. "processor plugin empty".

**Review by hariso — APPROVED** [review]
:muscle:

File: api/v1alpha/conduit_validator.go (hunk 1 above)
Comment (lyuboxa) [reply api/v1alpha/conduit_validator.go:28]: @hariso will do!

Comment (lyuboxa) [reply api/v1alpha/conduit_validator.go]: Fixed here 666dd957084c3c724b9b975fd5dc1cc173265f97

**Review by hariso — APPROVED** [review]

**Pull request merged**